ninja-core/src/site/site.xml

```diff
@@ -89,34 +90,29 @@
             <item name="Working with XML" href="./documentation/working_with_xml.html"/>
 
         </menu>
-        <menu name="Advanced topics">
-
-            <item name="Validation" href="./documentation/validation.html"/>
-            <item name="Content negotiation" href="./documentation/content_negotiation.html"/>
 
-            <item name="Uploading files" collapse="true" href="./documentation/uploading_files.html"/>
-
-            <item name="Scheduler" href="./documentation/scheduler.html"/>
-            <item name="Lifecycle" href="./documentation/lifecycle.html"/>
+        <menu name="Advanced topics">
+            <item name="Security, i18n and more" collapse="true" href="./documentation/html_templating/advanced_topics.html">
+                <item name="Security" collapse="true" href="./documentation/security/getting_started.html"/>
+                <item name="Internationalization" collapse="true" href="./documentation/internationalization.html"/>
+                <item name="Static assets" collapse="true" href="./documentation/static_assets.html"/>
+                <item name="Validation" href="./documentation/validation.html"/>                     
+                <item name="Uploading files"  href="./documentation/uploading_files.html"/>
+                <item name="Content negotiation"  href="./documentation/content_negotiation.html"/>                
+            </item>                     
 
             <item name="Working with relational DBs" collapse="true" href="./documentation/working_with_relational_dbs/jpa.html">
                 <item name="DB Migrations" href="./documentation/working_with_relational_dbs/db_migrations.html"/>
             </item>
 
-            <item name="Using the cache" collapse="true" href="./documentation/using_the_cache.html"/>
-
-            <item name="Internationalization" collapse="true" href="./documentation/internationalization.html"/>
-
-            <item name="Static assets" collapse="true" href="./documentation/static_assets.html"/>
-
-            <item name="Sending mail" href="./documentation/sending_mail.html"/>
-
-            <item name="Logging" href="./documentation/logging.html"/>
-
-            <item name="Custom routing" href="./documentation/custom_routing.html"/>
-
-            <item name="Servlet bridge" href="./documentation/servlet_bridge.html"/>
-
+            <item name="Debugging" collapse="true"href="./documentation/debugging.html">
+                <item name="Scheduler" href="./documentation/scheduler.html"/>
```

Comment:
A I mentioned "The overall tree needs to be restructured, which is an incremental process."

Next step would most possibly be:

- either "Debugging" menu entry is renamed to "Development", or a new document is introduced as top-level, named "General Development" (or similar).